fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/WindowTransformer.java

```diff
@@ -347,10 +354,25 @@ public static WindowOperator standardize(AnalyticExpr analyticExpr) {
                 return sortKey;
             }
         });
-
         return sortedGroups;
     }
 
+    private static boolean isPrefixHyperPartitionSet(List<Object> hyperSet, List<Object> subSet) {
```

Comment:
Don't get this method want to do what.

Reply:
BTW, partition by don't need column order match

Reply:
Determine whether a partition satisfies the distribution of another partition, and determine whether to add ExchangeNode